index.js

```diff
@@ -148,8 +149,22 @@ async function handleAlert(location, alertType) {
         fallbackMessage: `An alert to check on the bathroom at ${location.displayName} was not responded to. Please check on it`,
         fallbackToPhoneNumbers: location.fallbackNumbers,
         fallbackFromPhoneNumber: location.twilioNumber,
+        sirenParticleId: location.siren_particle_id,
       }
-      braveAlerter.startAlertSession(alertInfo)
+
+      if (location.siren_particle_id != null) {
+        try{
+          await axios.post('https://api.particle.io/v1/devices/location.siren_particle_id/start-siren', {
```

Comment:
Are you trying to get the location's `siren_particle_id` to be inserted in the middle of this URL? If so, JS has a cool syntax for that using backticks and `${}`. In this case, you probably want: 
```
`https://api.particle.io/v1/devices/${location.siren_particle_id}/start-siren`
```

More explanation if you'd like: https://www.w3schools.com/js/js_string_templates.asp